src/coreComponents/constitutive/permeability/ParallelPlatesPermeability.hpp

```diff
@@ -44,19 +44,10 @@ class ParallelPlatesPermeabilityUpdate : public PermeabilityBaseUpdate
                 arraySlice1d< real64 > const & permeability,
                 arraySlice2d< real64 > const & dPerm_dDispJump ) const
   {
-    // TODO: maybe move to this computation or have the possibility of choosing.
-//    real64 const perm  = newHydraulicAperture*newHydraulicAperture*newHydraulicAperture / 12.0;
-//    real64 const dPerm = newHydraulicAperture*newHydraulicAperture / 4.0;
-
-    real64 const perm = 0.25 * ( oldHydraulicAperture*oldHydraulicAperture*oldHydraulicAperture +
-                                 oldHydraulicAperture*oldHydraulicAperture*newHydraulicAperture +
-                                 oldHydraulicAperture*newHydraulicAperture*newHydraulicAperture +
-                                 newHydraulicAperture*newHydraulicAperture*newHydraulicAperture ) / 12;
-
-    real64 const dPerm  = 0.25 * ( oldHydraulicAperture*oldHydraulicAperture +
-                                   2*oldHydraulicAperture*newHydraulicAperture +
-                                   3*newHydraulicAperture*newHydraulicAperture ) / 12;
+    GEOS_UNUSED_VAR( oldHydraulicAperture );
 
+    real64 const perm  = newHydraulicAperture*newHydraulicAperture / 12.0;
+    real64 const dPerm = newHydraulicAperture / 6.0;
```

Comment:
we should make sure that the convergence of the hydraulic fracturing benchmarks does not get worse by doing this. If it does we may need a switch.

Reply:
Yes, we will see how it behaves.